nemo/collections/common/losses/smoothed_cross_entropy.py

```diff
@@ -48,7 +48,7 @@ def input_types(self):
         """Returns definitions of module input ports.
         """
         return {
-            "logits": NeuralType(('B', 'T', 'D'), LogitsType()),
+            "log_probs": NeuralType(('B', 'T', 'D'), LogitsType()),
```

Comment:
The change of "logits" => "log_probs" will break all other users of this class. Please revert.